examples/er-rest-example/project-conf.h

```diff
@@ -93,6 +96,10 @@
 #undef COAP_PROXY_OPTION_PROCESSING
 #define COAP_PROXY_OPTION_PROCESSING   0
 
+/* Turn of DAO ACK to make code smaller */
+#undef RPL_CONF_WITH_DAO_ACK
+#define RPL_CONF_WITH_DAO_ACK          0
```

Comment:
This is a mistake. It is a change from our work with DAO ACK for scalability and should not be included in this PR. I will remove it. Thanks!